docs/csharp/programming-guide/classes-and-structs/destructors.md

```diff
@@ -50,13 +50,17 @@ Destructors are used to destruct instances of classes.
   
 -   A destructor does not take modifiers or have parameters.  
   
- For example, the following is a declaration of a destructor for the class `Car`:  
+ For example, the following is a declaration of a destructor for the `Car` class.
   
  [!code-cs[csProgGuideObjects#86](../../../csharp/programming-guide/classes-and-structs/codesnippet/CSharp/destructors_1.cs)]  
+
+A destructor can also be implemented as an expression body definition, as the following example shows.
```

Comment:
The official language spec is going to standardize on 'finalizer' over 'destructor'. (Both had been used previously.

We should match that.